games-board/pychess/pychess-0.99.4.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{5,6} )
+PYTHON_REQ_USE="sqlite"
+
+inherit gnome2-utils xdg-utils distutils-r1
+
+DESCRIPTION="A chess client for Gnome"
+HOMEPAGE="http://pychess.org/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gstreamer"
+
+DEPEND="
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	gnome-base/librsvg:2
+	x11-libs/gtksourceview:3.0
+	x11-libs/pango
+	x11-themes/adwaita-icon-theme
+	gstreamer? (
+		dev-python/gst-python:1.0
```

Comment:
doesn't this also need `[${PYTHON_USEDEP}]`?

Reply:
also, have you tried `EAPI=7`?

Reply:
Added both. Thanks.